platform/core/src/studies/services/wado/retrieveMetadataLoaderAsync.js

```diff
@@ -66,12 +66,16 @@ function makeSeriesAsyncLoader(
 export default class RetrieveMetadataLoaderAsync extends RetrieveMetadataLoader {
   configLoad() {
     const { server } = this;
-
-    const client = new api.DICOMwebClient({
-      url: server.qidoRoot,
-      headers: DICOMWeb.getAuthorizationHeader(server),
-    });
-
+	
+	
+	var options = {
+		url: server.baseUrl,
+        headers: DICOMWeb.getAuthorizationHeader(server),
+		qidoURLPrefix: server.qidoURLPrefix,
+		wadoURLPrefix: server.wadoURLPrefix,
```

Comment:
Does it make more sense to not have a baseUrl, and instead specify the `qidoRoot` and `wadoRoot`? We even already have keys for these. What value does this more granular option setting buy us?